lib/menu.rb

```diff
@@ -0,0 +1,13 @@
+class Menu
+
+  def dish(dish,price)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `dish`. If it's necessary, use `_` or `_dish` as an argument name to indicate that it won't be used. You can also write as `dish(*)` if you want the method to accept any arguments but don't care about them.<br>Lint/UnusedMethodArgument: Unused method argument - `price`. If it's necessary, use `_` or `_price` as an argument name to indicate that it won't be used. You can also write as `dish(*)` if you want the method to accept any arguments but don't care about them.